network-configuration/jclouds-partitiongroup/pom.xml

```diff
@@ -23,6 +23,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
         <jclouds.version>1.9.1</jclouds.version>
+        <hazelcast.version>3.12.6</hazelcast.version>
```

Comment:
I think we have to keep jclouds examples at 3.12 because we haven't yet released the plugin compatible with 4.0 - https://github.com/hazelcast/hazelcast-jclouds/releases. Needs to be double-checked though.

Reply:
Yes, it's like that. We can't upgrade that yet.